File: helfomat-infrastructure-typo3/src/main/java/de/helfenkannjeder/helfomat/infrastructure/typo3/domain/TGroupTemplate.java
package de.helfenkannjeder.helfomat.infrastructure.typo3.domain;

import javax.persistence.Entity;
import javax.persistence.Id;

/**
 * @author Valentin Zickner
 */
@Entity(name = "tx_helfenkannjeder_domain_model_grouptemplate")
public class TGroupTemplate {

    @Id
    private long uid;

    private String name;

    private String description;

    private String suggestion;

    public long getUid() {
        return uid;
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public String getSuggestion() {
        return suggestion;
    }

}

File: helfomat-infrastructure-typo3/src/main/kotlin/de/helfenkannjeder/helfomat/infrastructure/typo3/domain/TGroupTemplate.kt
package de.helfenkannjeder.helfomat.infrastructure.typo3.domain

import javax.persistence.Entity
import javax.persistence.Id

/**
 * @author Valentin Zickner
 */
@Entity(name = "tx_helfenkannjeder_domain_model_grouptemplate")
data class TGroupTemplate (
    @Id
    var uid: Long,
    var name: String?,
    var description: String?,
    var suggestion: String?

)